### PR TITLE
About billing and sponsorship payments

### LINKED ISSUE #33785: No reference to PayPal end-of-support in entire directory tree of GitHub Sponsors program
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/.github/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

[About payment methods for sponsorships](https://docs.github.com/en/sponsors/sponsoring-open-source-contributors/about-sponsorships-fees-and-taxes#about-payment-methods-for-sponsorships)

### What part(s) of the article would you like to see updated?

Two weeks ago, I decided I'd sponsor the owner of the repository where I spend most of my time online. Not knowing what billing methods were supported and hoping there was PayPal support, I was relieved when I saw the PayPal button in my user settings. However, since I hadn't used my wallet for a long time, regaining access to the account turned out to be rather challenging.

Actually, you will not believe the trouble I went through until I eventually managed to access the account. It's mine, don't worry. Practically everything that could go wrong, did. Password recovery, two-factor authentication resets, dying phone refusing to charge, relinking financial institution, etc. Hours later and all set up, what'd I see? PayPal's no longer supported and the only announcement was the next image.

image

Correct, it was announced in the changelog but that ain't the first place to go looking for that kind of notice. The announcement's already preceded by dozens of other posts. Let it be said that the general user would assume PayPal to be functional in the event of no or minimal triggers.

What astonished me most was the lack of the PayPal company name in the entire directory tree of the program, especially since it's not dead and only affects GitHub Sponsors. Until now, and I went for it straight away because I'm positive that many users encountered similar situations before.

### Additional information

The occurrence frequency is permanent. The issue will direct to a PR momentarily, and vice-versa.

### PR DESCRIPTION
**Pull request by Yewzir**
### Why:

Fully spontaneous, with reasoning and closes #33785. My PR renders an updated billing and payment section on GitHub Sponsors where PayPal end-of-support is explicitly mentioned, plus a bit of free advertising for GitHub products that are actually still supported, I think. I'm not an employee so if one of the products isn't supported or has never been supported by PayPal funding, I'm happy to hear and I will perform the respective modifications in turn. I can't think of a better place to sneak that in while it's still actually relevant information.

Not to worry, I will not propose being able to set you up with the translation in the Dutch language. Well, I could but I won't. Just because I noticed _translation_ a lot in the guidelines and stylings, I gave it a spin on GT and I'm definitely amazed at how accurate the translation it generates is.

I think it's a decent looking, non disturbing block and a smooth read. Translations, those are the least of my worries. So, here goes.

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Free, Pro & Team Editions

image

Enterprise Cloud & Server Editions

image

### Check off the following:

- [X] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [X] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by Yewzir** [comment]
Now I have finished, I swear.

**Review by Yewzir — COMMENTED** [review]
Almost finished, I swear.

**Review by Yewzir — COMMENTED** [review]
This reflects all changes leading to the images in the PR description for the different editions.

**Comment by nguyenalex836** [comment]
@Yewzir Thanks so much for opening a PR! I'll get this triaged for review ✨

File: content/sponsors/sponsoring-open-source-contributors/about-sponsorships-fees-and-taxes.md [hunk 1]
```diff
@@ -37,23 +37,30 @@ If the account you want to sponsor does not have a profile on {% data variables.
...
+**Note:** When sponsoring through GitHub, your balance will be charged effective immediately. {% data reusables.sponsors.prorated-sponsorship %}
 
 {% endnote %}
 
-Organizations can also pay for sponsorships by invoice. For more information, see "[AUTOTITLE](/sponsors/sponsoring-open-source-contributors/paying-for-github-sponsors-by-invoice)."
+Alternatively, enterprises and organizations may opt to be billed by recurring invoice. For additional information on billing by invoice, please click on [AUTOTITLE](/sponsors/sponsoring-open-source-contributors/paying-for-github-sponsors-by-invoice).
+
+{% warning %}
+
+**Warning:** As of February 23, 2023, GitHub Sponsors does not longer support sponsorships using PayPal. Unfortunately, sponsoring the GitHub community using PayPal wallets is now impossible. While this only affects GitHub Sponsors, please note that it is still possible to fund GitHub Pro, GitHub Copilot, Actions and Packages, Storage, Codespaces and Git LFS Data using PayPal, however.
```
Comment (andimiya) [inline content/sponsors/sponsoring-open-source-contributors/about-sponsorships-fees-and-taxes.md:57]: Small typo: `GitHub Sponsors does not longer support sponsorships using PayPal.`

**Review by andimiya — COMMENTED** [review]
Just a small typo/grammar suggestion but looks good, thank you!

File: content/sponsors/sponsoring-open-source-contributors/about-sponsorships-fees-and-taxes.md (hunk 1 above)
Comment (Yewzir) [reply content/sponsors/sponsoring-open-source-contributors/about-sponsorships-fees-and-taxes.md:57]: @andimiya 

Continuously reading the same thing makes one stop reading eventually 😅

**Review by nguyenalex836 — APPROVED** [review]

**Comment by nguyenalex836** [comment]
@Yewzir Thank you for updating your PR with the suggestions @andimiya provided! I'll get this merged 🍏

**Comment by Yewzir** [comment]
> @Yewzir Thank you for updating your PR with the suggestions @andimiya provided! I'll get this merged 🍏

Nice, thank you!

**Comment by Yewzir** [comment]
@nguyenalex836 

I hate to revive closed material, but it was purely coincidential that I noticed a revert request, already. Eh you're not actually telling me y'all go through such a streamlined contribution process, whereas one contribution requires more time than another, to have it reverted afterwards while we had all this time to settle things. Not really following here, twenty-five minutes between merge and revert request to be precise. No, not following, at all.

**Pull request merged**